temoa/extensions/monte_carlo/mc_worker.py

```
@@ -70,48 +69,63 @@ def run(self):
+                    except (ValueError, AttributeError):
+                        # Key not defined in ConfigDict, skip it
+                        pass
+
+                # Handle solver log path for appsi
+                if self.solver_log_path:
+                    log_location = (
+                        self.solver_log_path
+                        / f'solver_log_{self.worker_number}_{self.solve_count}.log'
+                    )
+                    try:
+                        setattr(self.opt.config, 'log_file', str(log_location))
```

Ruff (0.14.10)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)